Assets/Scripts/Game/WeaponManager.cs

```diff
@@ -481,7 +481,22 @@ private void MeleeDamage(FPSWeapon weapon)
 
                 // TODO: Use correct damage based on weapon and swing type
                 // Just using fudge values during development
```

Comment:
I think Interkarma probably won't be around for a few hours. But I agree it probably would be fine to add data tables and I expect Interkarma will agree, too. I believe metal materials are already implemented, so  it might be fairly easy to just use the base damage values and add in the material modifier. I don't know if there's any information out there about how exactly the different swing types affect the damage type, though.